analyzers/its/expected/akka.net/Akka--netstandard2.0-S2259.json

```diff
@@ -1,17 +0,0 @@
-{
-"issues":  [
-{
-"id":  "S2259",
```

Comment:
This was FP. We now know that `numberOfExtraArgs > 0` means `extraArgsValues` is `NotNull` and don't raise.